siliconcompiler/report/Home.py

```diff
@@ -0,0 +1,601 @@
+import streamlit
+from streamlit_agraph import agraph, Node, Edge, Config
+from streamlit_tree_select import tree_select
+from streamlit_toggle import st_toggle_switch
+from PIL import Image
+import os
+import argparse
+import json
+import pandas
+from siliconcompiler.report import report
+from siliconcompiler import Chip, TaskStatus
+from siliconcompiler import __version__ as sc_version
+
+
+success_color = '#8EA604'  # green
+pending_color = '#F5BB00'  # yellow, could use: #EC9F05
+failure_color = '#FF4E00'  # red
+
+inactive_toggle_color = "#D3D3D3"
+active_toggle_color = "#11567f"
+track_toggle_color = "#29B5E8"
+
+# TODO: change to accomodate different operating systems
+sc_logo_path = os.path.dirname(__file__) + "/../data/logo.png"
+
+sc_about = [
+    f"SiliconCompiler {sc_version}",
+    '''A compiler framework that automates translation from source code to
+     silicon.''',
+    "https://www.siliconcompiler.com/",
+    "https://github.com/siliconcompiler/siliconcompiler/"
+]
+
+streamlit.set_page_config(page_title="SiliconCompiler",
+                          page_icon=Image.open(sc_logo_path),
+                          layout="wide",
+                          menu_items={
+                              "Get help": "https://docs.siliconcompiler.com/",
+                              "Report a Bug": '''https://github.com/siliconcomp
+                              iler/siliconcompiler/issues''',
```

Comment:
this should probably be reformatted to keep the URL together. You could do a newline at `page_title` to avoid as much white space.